src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -249,7 +249,7 @@ internal static int CombineHashCodes(int h1, int h2)
             // Forward to helper class in Common for this
             // We keep the actual hashing logic there, so
             // other classes can use it for hashing
-            return HashHelpers.Combine(h1, h2);
+            return HashHelpers.Combine(HashHelpers.Combine(h1, h2), HashHelpers.RandomSeed);
```

Comment:
@stephentoub I refactored things so the random seed is only combined once (at the beginning of the hash).